src/webview/cluster/app/clusterView.scss

```diff
@@ -79,3 +79,27 @@
 .buttonSecondary:disabled {
     opacity: 0.4 !important;
 }
+
+.buttonRed {
```

Comment:
instead of modifying the button through the `.scss`, take a look at [`color='error'`](https://mui.com/material-ui/react-button/#color) and see if that's sufficient. That way you don't need to modify the button for all the different states.

Reply:
![image](https://github.com/redhat-developer/vscode-openshift-tools/assets/73968480/f5222acf-f36e-46e2-9f7b-53b8f109b638)

It's a bit hard to see when it's disabled, does this work?

Reply:
I think stick with what you had.

If we were managing themes using `<ThemeProvider>` in this webview, I think it could look better, but that would take a lot more refactoring work.